src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/Modals/ResetPasswordModal.js

```diff
@@ -0,0 +1,37 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Row,
+  Col,
+} from '@folio/stripes/components';
+
+import withCopyModal from './WithCopyModal';
+
+const ResetPasswordModalBody = ({ email, name }) => {
```

Comment:
In case if we agreed on using multiline destructuring, please, move `email` and `name` to separate lines